profile/CSIP.xml

```diff
@@ -176,8 +176,8 @@
             </requirement>
             <requirement ID="CSIP3" REQLEVEL="SHOULD" RELATEDMAT="VocabularyType" EXAMPLES="metsRootElementExample1 metsRootElementExample2 metsRootElementExample3">
                 <description>
-                    <head>Other general content type</head>
-                    <p xmlns="http://www.w3.org/1999/xhtml">The @csip:OTHERTYPE attribute must be used for stating the general type of the package (genre) when @TYPE has the value "OTHER"</p>
+                    <head>Extended content type</head>
```

Comment:
Maybe `Custom content type`? "Extended content type" sounds a bit like "Type of extended content," as if we had "normal" content and "extended" content. Although, "custom" suffers from the same problem as "Type of custom content."

But essentially it is custom, as the attribute is for recording a custom type-description value that is not present in the official vocabulary.